integrationtest/src/it/oas2SkipOnError/verify.groovy

```diff
@@ -0,0 +1,4 @@
+String log = new File(basedir, "build.log").text
+
+assert log.contains("[INFO] Start creation of KieBase: defaultKieBase")
```

Comment:
you don't need to assert on the "KieBase" log entry bc it isn't a feature we need to have (more a side-effect, which we might better disable in logging settings).

Same applies for other verify.groovy scripts